tests/func/test_query.py

```diff
@@ -80,23 +80,23 @@ def get_latest_job(
 def test_query_cli(cloud_test_catalog_tmpfile, tmp_path, catalog_info_filepath, capsys):
     catalog = cloud_test_catalog_tmpfile.catalog
     src_uri = cloud_test_catalog_tmpfile.src_uri
+    catalog.create_dataset_from_sources("animals", [src_uri], recursive=True)
 
-    query_script = f"""\
+    query_script = """\
     from datachain.query import DatasetQuery
     from datachain import C
     from datachain.sql.functions.path import name
 
-    DatasetQuery({src_uri!r}, catalog=catalog).mutate(name=name(C.path)).save("my-ds")
+    DatasetQuery("animals", catalog=catalog).mutate(
+        name=name(C("file__path"))
+    ).save("my-ds")
     """
     query_script = setup_catalog(query_script, catalog_info_filepath)
 
     filepath = tmp_path / "query_script.py"
     filepath.write_text(query_script)
 
     query(catalog, str(filepath))
-    out, err = capsys.readouterr()
-    assert not out
-    assert not err
```

Comment:
Why was this removed?

Reply:
Actually nothing should be captured because we don't capture output at all and not because of the reason I mentioned above (both approaches print stuff to stdout). 
I needed to move listing command (done with `catalog.create_dataset_from_sources(...)`) to script code itself, not outside, and now these lines of code can be returned and it works fine.